internal/trace/transport.go

```diff
@@ -34,6 +37,9 @@ var (
 	}
 )
 
+// payloadSizeLimit limits the maximum size of the response body to be printed.
+const payloadSizeLimit int64 = 4 * 1024 * 1024 // 4 MiB
```

Comment:
Do you really want to print 4 MiB of data? not 4 KiB?

Reply:
4 KiB actually makes sense. A text of 4 KiB size looks like:


> duqmfjqvjwnubdtjnrwxueflzbbvslkvwqsppuamnnjqzfcocpyegelcetinpprwgtqajphyyvjytpuixwftrtflczzdtjlrwanuspwkweqncnosskiowgfzsefcjpptxjsoyfvbtgmwtojbduqioftzhavdrgqzazvpbuwnpacyrztfcewwhjcmozpegmcgmcepzuvbzvobxrefyypgsrqbfgzaizwteiyooaitjieuqnzedklfdjvmfjyowpzoxktagmoeszxyhkxjekvougwwwvsubcmirbdkmtmhpodjagdrbcrnyilhexnbeocoyqfhfxrpvzsthakubnyolkhmwujfnhpzshrwfadxaawnhvmljtzbcqgsjxbebffxcvriaykwtbyrfzvtilfwxjfcdsxgtjfqmfggvzmywgeorzxzzugdbqulmwnbynboedzyhfwscimwswpvwrpfxtgyntfzefuymyndjwjqhuztwzuggpiuubmalfadlhoybhhpyluyredvqbtnogserpymfqyryqnxdpdftrigoraomutvdfpdqfkbwgoahjcrvgwkdtnkvbuiuolokaslryashbctdwrtquwpwbpeobtlxiywjnwncfoiylvtlelvsqbwocvzacvvizupbtzagggurcqvrhwzdxagfeewixsrnglpjuendbocpvresgnzxlslrhwxhcbhfmiqzzbcluorrkpsedqiqljiiwsfzzcdaduukzkxmcmweewtxeukyguuvpgrrpuxmfswtezcfeyxojdzvlaodzrgngikiazresddmmuxdnfjsrqtjcdxrfluhnmnkcfzsrckdndwejmakprkexovosljkhlorwzwpcuxbuwtbzvzfvtsihyxdelywtwlygwsjtercpbhajxxwvdxjdsgjrfrcdduaxjfsmkudnpberopjonasztxaxwybnvkginkosnbocthkdssblmxbopwnfvhqoqpqwvvmwqkstzjbsducgkwnqxuecokliggqhwheapkvmdqqviyaugotxqjfdbvchwriuutirrtmhzktqhiuoxlfkeovnwffetdbcugjvyudxlkmofwiaqccdtexspwkhvnldaqngjjmerwcotscluxqnanslauugcqqjdsdldrmszsnjerznnabaeazncvzmbiixtownjarkalctnccdcjwpnygpxzhtwkwfosikmlentngsixvvgfxfxzygliencqgwpkodouodrvbmipdueskyqtqbdsjviicsgkbdkcnhfoahioguunhbpqmrnfssywsfsiyowrnhnrvhwwqcwwxmgezjmjelcdfgsgfandaabaeptyhudpykwggwirtmteirdopvdfidahhnohgwjsypbiadqmexcbsnwjjquvzdtimmrusuxwhcviygcngjsifoxiqvjxyliyprtfjyvhncbhpyciysoopytsetotphebxtultafyoefxtxypgtjstiuqpvfkptziqrwhcgxlxyiusbevnylmhjiwbkmzwcslyepgvimqthxxzitqkyggecxaghdqbfrwvqkpxvbupvsxomeommuumrnnfhtypawgdvefrruizokqfblulrejmpsrwqsxdtsqkskrnslemrmdkfruzlrtvlnbpngltacoaxsxhrsibvobfnezkombvbjartfnwrzpsikzulvxrjdrajdulhxawfbigqewuascuemixrcbuytkderrrjsycehmawkanazhdkuhyqzlwrvxmangprpabokgzpawhptvuiotfohxvfabblcksttqnxwtybmvpyjlbcuuoaeowurcvolopccauscjqvubvuqfvedttlwlgsdkdyoqbaphqqevsnnwjghzztgwbtpczvhycqqfluculquandvuuazbohahmuzyusnjqaiohwxfqmkmmzxnvnqbxmcnqynpujptndeegtcojdesxwcayiawtszczfcbmjnnnmrldqybntvsjenqezffzpuldstugzpmnglmtxbgionsfxlxshddbdyvjarhftnrcdtlexjbsrqmhflxwmikifozeotcvqwmihneahtxhsoaejpegvilrfffqcgpcbytpxvmkhldbptckxjpvmtqyiyhzhszygdrqdbgethowszlwpsihdowryhqhggowmigzbcehppshcykdtwgpmykfdwndktyuunhuhvcujjsvhxbkezzkmjbkxsjhhynktokslbfbxjvfwgleqjdfertcaabkusvzyqdzitksusduyndiknzzreuvmsqlpkhwcnslbjlsicrnqmlsqcgddqccuolylpmjrvasvyixuyemkzzajpynfnexuzbtaumvdqfwoywimpbfeblquykvhvntyorfgmfoaaqsukkehjbjefmdwqemwfcruiotfdhddjpkfthcuuxbqoruxhdpzlppxeaarwfbpkvwmzjdjprhupjbydlumniqiuiztpijyqughmtmlgaotfyxttvvrsmearqfiwkqmklytpydidqvjeyovfhhwntqokogdxuweuzsizzuqjaiwbyilmbbwsvmejuscdgzmrjoyijvqsisnnqrdofxrcnyjfvvhxdbwtkzexueslhzwnptnoxfxvausjutdaswdpxjivcloglbilgfldfbeualgbonocdoofzyfvgepkzcajwtljccsurgiavudmvwstardpboxknioswfggnqtgdilmatxrotqxwoveutgemvzcrzviaqombszwlvrmlfjthwtqenjgzfbfnsxnkrzzlipszbzaebwmimuqdqhaswdnxtwewfmjmpkuppcbdeqesgkozrcwvzkeeeeduzsxblxdmeyewgorlchivqbvoputtihblbqovyqpxkqjieadqbtboywkyzxtmiebtmaospbjniiydbsclwmwyjadxcqxfdylclrcynmtutegebahiwsuiifmyxbmahqmfdcqtuohqjbcgvchiqtstocvfdiemspnugpxfkenkwviagwftsimgfpxzqprrzhftyxtfcjwwihqcnexfcggklnadixxfojrhsulunrtwhxkhcbasueufzwyfeokrocpqhpetoxswxxlyjbcazorzlmjhfdnbotvdqybcxzzqmdsyjxvmehmbewbhmbogrbnnrmjzwwqodzdigeccmwlxtqxctfchndtovtoeiuvogsfyslyfvrqzcjgjvuavhoalvhdzantqpyxnnrfyxnthjelekojivxmkztrtmofasjtkjmhpsbbnacuypnrjzhtunknebjlccqvnnlbnzfudmedmhcdcimpojlhzfuctewrioncxejdixmzwiofeketrxyhuotjbqjdpdijrvrygvjrqmieurmtlhpdszojosjmpxdzypdyegmwwhgswihovwxeoddgujfyedxhqzswoqrrtvjxulohjoycwuujatrvzdtooarmdpgiirbgdwnyvzysdezyeohumbcxplbajdhgbxobsvkdshfkgfpdimluakeayitdjxpycqdkjqjuoduglnglhnxupisxfynfrmoccpetouupbywzclvakgbkemblaxiicfacdstyjsnvuoepltuwvtnaqjmmwtzctdzxopmahdueocgihtajrgexyqaxlctbghyfpslxbkdtgzckvzjwgoazfbsnktfncvjekczmohangyuggyvlcateaikczmwwcqgdyzlzmhhpbhkaubtozfspzinctdjcfljtjafunbahtvtroojwjdgrnysupsflqkjvdrmikimeaqlfzcgpdymjwnpjawehcrqgfgrxdmmltpygwsvflbzotvzidajcfvdkkwlytapxzsexttegvnrxxflfyccamgnktnrrzjccjcbsazumvudhbarhfzssbwbfswlbdzjirqbxurbrysdaezchjejkcpxfkcnshpmzybfzljkpnkaxjzbjzaairyaiohyjwafcufgayxwviejvewtcobtjxtvliavbetepkegerzrgnougkswwc